build.gradle.kts

```diff
@@ -31,33 +20,11 @@ allprojects {
     }
 }
 
-apply(plugin = "maven-publish")
-apply(plugin = "signing")
-
 subprojects {
     if (name.contains("benchmark")) {
         return@subprojects
     }
-
-    repositories {
-        mavenCentral()
-    }
-
-    apply(plugin = "maven-publish")
-    apply(plugin = "signing")
-
-    publishing {
-        repositories {
-            configureMavenPublication(project)
-        }
-
-        val javadocJar = project.configureEmptyJavadocArtifact()
-        publications.withType(MavenPublication::class).all {
-            pom.configureMavenCentralMetadata(project)
-            signPublicationIfKeyPresent(project, this)
-            artifact(javadocJar)
-        }
-    }
+    apply(plugin = "publish-conventions")
```

Comment:
may be apply it directly in `multplatform-lib-convention`?

Reply:
I'm not sure that only multiplatform artifacts will be published in the future.

@fzhinkin, if so, then we may think about merging.

Reply:
Yeah, I understand that.
In my own lib I use separate convention plugins for basic configuration ([like common compiler options](https://github.com/whyoleg/cryptography-kotlin/blob/main/build-logic/src/main/kotlin/buildx-multiplatform.gradle.kts)), targets configuration ([JVM](https://github.com/whyoleg/cryptography-kotlin/blob/main/build-logic/src/main/kotlin/buildx-target-jvm.gradle.kts), [JS](https://github.com/whyoleg/cryptography-kotlin/blob/main/build-logic/src/main/kotlin/buildx-target-js.gradle.kts), [combined](https://github.com/whyoleg/cryptography-kotlin/blob/main/build-logic/src/main/kotlin/buildx-target-all.gradle.kts), and etc) and publishable modules ([with documentation and publication](https://github.com/whyoleg/cryptography-kotlin/blob/main/build-logic/src/main/kotlin/buildx-multiplatform-library.gradle.kts)).
So they could be combined then in modules: [one](https://github.com/whyoleg/cryptography-kotlin/blob/main/cryptography-core/build.gradle.kts), [two](https://github.com/whyoleg/cryptography-kotlin/blob/main/cryptography-providers/webcrypto/build.gradle.kts), [three](https://github.com/whyoleg/cryptography-kotlin/blob/main/test-support/build.gradle.kts)
While it could be overhead for just 2 modules, which we currently have, it's very scalable to new modules with different targets supported and also removes usages of `subprojects` in root module

Reply:
> I'm not sure that only multiplatform artifacts will be published in the future.

I'm not sure either. Can easily imagine a separate module providing integration with some platform specific API or library.

Reply:
For now, we'll leave a separate plugin for publishing, and a separate one for a multiplatform project